src/index.html

```diff
@@ -917,37 +919,9 @@ <h5>Archive process</h5>
                         </div>
                         <div class="tabs-panel is-active" id="accessionPanel">
                             <ul class="accordion" data-accordion>
-                                <h3 class="padding-bottom-large">EVA & dbSNP Transition</h3>
-                                <div>
-                                    <p>
-                                        EMBL-EBI will maintain reliable accessions for non-human genetic variation data through the European Variation Archive (EVA). NCBI's dbSNP database will continue to maintain stable identifiers for human genetic variation data only. This change will enable a more rapid turnaround for data sharing in this burgeoning field.

```

Comment:
I guess we might want to mention this somewhere on some future "About us" page, but I definitely agree it doesn't belong here and we shouldn't talk about it as a "transition" anymore.